content/en/opentelemetry/setup/ddot_collector/custom_components.md

```diff
@@ -1,16 +1,13 @@
 ---
 title: Use Custom OpenTelemetry Components with Datadog Agent
-private: true
+aliases:
+- "/opentelemetry/setup/ddot_collector/custom_components/"
```

Comment:
```suggestion
- "/opentelemetry/agent/agent_with_custom_components"
```

Reply:
I think this might be the alias redirect you meant to add, since the existing redirect already points to the same file.